src/bin/pgcopydb/cli_clone_follow.c

```diff
@@ -47,6 +47,7 @@
 	"  --skip-collations             Skip restoring collations\n" \
 	"  --skip-vacuum                 Skip running VACUUM ANALYZE\n" \
 	"  --skip-db-properties          Skip copying ALTER DATABASE SET properties\n" \
+	"  --skip-ctid-split             Skip spliting tables by ctid\n" \
```

Comment:
I think `--skip-split-bt-ctid` would be a improvement for the naming, what do you think?